### PR TITLE
Fix unable to restore due to missing MicroBuild props/targets

### PR DESCRIPTION
**Pull request by davkean**

### CODE REVIEW

**Comment by davkean** [comment]
tag @dotnet/project-system @basoundr @333fred

File: build/Targets/VSL.Imports.targets [hunk 1]
```diff
@@ -2,12 +2,20 @@
...
+    <MicroBuildPackageDirectory>$(NuGetPackageRoot)\MicroBuild.Core\$(MicroBuildVersion)\</MicroBuildPackageDirectory>
+  </PropertyGroup>
+
+  <!-- During Restore the props/targets won't be present-->
+  <ImportGroup Condition="'$(ProjectSystemLayer)' != 'Dependency' AND '$(NonShipping)' != 'true' AND Exists('$(MicroBuildPackageDirectory)')">
+    <Import Project="$(MicroBuildPackageDirectory)\build\MicroBuild.Core.props" />
+    <Import Project="$(MicroBuildPackageDirectory)\build\MicroBuild.Core.targets" />
+  </ImportGroup>
+
   <!-- With UseCommonOutputDirectory turned on, any copy-local projects/references break
```
Comment (333fred) [inline build/Targets/VSL.Imports.targets]: What's this comment? Is this relevant to the line below after the change?

Comment (davkean) [inline build/Targets/VSL.Imports.targets]: It's relevant to the code below it - the Imports were put in between it.

**Comment by srivatsn** [comment]
👍
